app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script.rb

```diff
@@ -21,6 +21,10 @@ def create_in_provider(manager_id, params)
       task_ids.each { |tid| MiqTask.wait_for_taskid(tid) }
 
       find_by!(:manager_id => manager.id, :manager_ref => job_template.id)
+    rescue AnsibleTowerClient::Error => err
+      message = JSON.parse(err.message) rescue nil
+      raise if message.nil?
+      raise message.values.flatten.join(", ")
```

Comment:
If we do hit the inline rescue would this code not just re-raise that error, instead of the initial `AnsibleTowerClient::Error` that we really want to raise?

I think this is more what you want:
```ruby
rescue AnsibleTowerClient::Error => err
  begin
    raise JSON.parse(err.message).values.flatten.join(", ")
  rescue JSON::ParserError
    raise err
  end
end
```